lib/puppet/provider/selinux_fcontext/semanage.rb

```diff
@@ -30,7 +30,7 @@ def self.prefetch(resources)
     execpipe("#{command(:semanage)} fcontext -lC") do |out|
       lines = out.readlines[2..-1] || []
       lines.each do |line|
-        filespec, filetype, context = line.split(/\s\s+/)
+        filespec, filetype, context = line.scan(/^(\S+)\s+(\S+\s?\S+)\s+(\S+)\s+$/).flatten
```

Comment:
This will not work properly in real scenario because of the _$_ character at the end of the regex. The _line_ has format like below:
```
"/tmp/somepath(/.*)?                                 all files          system_u:object_r:httpd_sys_content_t:s0 \n"
```
and the _\n_ makes a problem with the _$_. Remove it and it will work.